CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
 # Changelog
 
-
 This changelog is auto generated by scanning the commit history.
-Commits have to follow following convention: https://www.conventionalcommits.org/en/v1.0.0/
+Commits have to follow following convention: <https://www.conventionalcommits.org/en/v1.0.0/>
```

Comment:
Something went wrong here.... Why is the changelog touched? I guess you need to change something in your github settings :)

Reply:
Changelog has been fixed.